docs/sce_file_contents.md

```diff
@@ -170,32 +177,32 @@ The list of highly variable genes used was selected using `scran::modelGeneVar`
 The following command can be used to access the PCA results:
 
 ```r
-reducedDim(sce, "PCA")
+pca <- reducedDim(sce, "PCA")
```

Comment:
I don't know how I feel about the assignment additions. I feel like the bare command was sufficient and probably preferred in many contexts.